kitsune/wiki/views.py

```diff
@@ -166,6 +166,11 @@ def document(request, document_slug, template=None, document=None):
     else:
         document_css_class = ''
 
+    if request.MOBILE and 'minimal' in request.GET:
```

Comment:
I think this should be available even when the request isn't from mobile. Embeddable versions of SUMO seem generally useful. If we don't do that, we should also make sure that BuddyUp's embedding method is properly marked as mobile.